### PR TITLE
conjure: set branch to master; add vim-oscyank, nvim-scrollview

### PR DESCRIPTION
**Pull request by onsails**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by onsails** [comment]
Result of `nixpkgs-review pr 106281` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>9 packages built:</summary>
  <ul>
    <li>vimPlugins.coc-nvim</li>
    <li>vimPlugins.fzf-vim</li>
    <li>vimPlugins.julia-vim</li>
    <li>vimPlugins.nerdtree-git-plugin</li>
    <li>vimPlugins.nvim-highlite</li>
    <li>vimPlugins.nvim-treesitter</li>
    <li>vimPlugins.vim-clap</li>
    <li>vimPlugins.vim-oscyank</li>
    <li>vimPlugins.vimtex</li>
  </ul>
</details>

File: pkgs/misc/vim-plugins/vim-plugin-names [hunk 1]
```diff
@@ -394,6 +394,7 @@ nvim-treesitter/nvim-treesitter-textobjects
 nvim-treesitter/playground
 ocaml/vim-ocaml
 octol/vim-cpp-enhanced-highlight
+ojroques/vim-oscyank@main
```
Comment (jonringer) [inline pkgs/misc/vim-plugins/vim-plugin-names:397]: is the `@main` needed?

Comment (onsails) [reply pkgs/misc/vim-plugins/vim-plugin-names:397]: yes, without that `update.py` fails with 404 atom feed request.

**Comment by onsails** [comment]
Result of `nixpkgs-review pr 106281` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>41 packages failed to build:</summary>
  <ul>
    <li>spacevim</li>
    <li>vimPlugins.Coqtail</li>
    <li>vimPlugins.Spacegray-vim</li>
    <li>vimPlugins.YouCompleteMe</li>
    <li>vimPlugins.airline (vimPlugins.vim-airline)</li>
    <li>vimPlugins.coc-explorer</li>
    <li>vimPlugins.coc-nvim</li>
    <li>vimPlugins.completion-nvim</li>
    <li>vimPlugins.conjure</li>
    <li>vimPlugins.defx-git</li>
    <li>vimPlugins.deol-nvim</li>
    <li>vimPlugins.jedi-vim</li>
    <li>vimPlugins.lean-vim</li>
    <li>vimPlugins.lh-brackets</li>
    <li>vimPlugins.nvim-gdb</li>
    <li>vimPlugins.nvim-highlite</li>
    <li>vimPlugins.nvim-lspconfig</li>
    <li>vimPlugins.nvim-lsputils</li>
    <li>vimPlugins.nvim-scrollview</li>
    <li>vimPlugins.nvim-treesitter</li>
    <li>vimPlugins.nvim-treesitter-context</li>
    <li>vimPlugins.nvim-treesitter-textobjects</li>
    <li>vimPlugins.polyglot (vimPlugins.vim-polyglot)</li>
    <li>vimPlugins.telescope-nvim</li>
    <li>vimPlugins.vim-clap</li>
    <li>vimPlugins.vim-floaterm</li>
    <li>vimPlugins.vim-go</li>
    <li>vimPlugins.vim-hexokinase</li>
    <li>vimPlugins.vim-illuminate</li>
    <li>vimPlugins.vim-javacomplete2</li>
    <li>vimPlugins.vim-lsp</li>
    <li>vimPlugins.vim-ocaml</li>
    <li>vimPlugins.vim-oscyank</li>
    <li>vimPlugins.vim-test</li>
    <li>vimPlugins.vim-visual-multi</li>
    <li>vimPlugins.vim-vsnip</li>
    <li>vimPlugins.vimsence</li>
    <li>vimPlugins.vimspector</li>
    <li>vimPlugins.vimtex</li>
    <li>vimPlugins.vista-vim</li>
    <li>vimPlugins.yats-vim</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 106281` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
40 packages built:
</summary>
<ul>
<li>spacevim</li>
<li>vimPlugins.Coqtail</li>
<li>vimPlugins.ale</li>
<li>vimPlugins.awesome-vim-colorschemes</li>
<li>vimPlugins.brainfuck-vim</li>
<li>vimPlugins.coc-explorer</li>
<li>vimPlugins.coc-nvim</li>
<li>vimPlugins.completion-nvim</li>
<li>vimPlugins.conjure</li>
<li>vimPlugins.defx-nvim</li>
<li>vimPlugins.deol-nvim</li>
<li>vimPlugins.fern-vim</li>
<li>vimPlugins.jedi-vim</li>
<li>vimPlugins.lean-vim</li>
<li>vimPlugins.lh-brackets</li>
<li>vimPlugins.ncm2-jedi</li>
<li>vimPlugins.nvim-dap</li>
<li>vimPlugins.nvim-gdb</li>
<li>vimPlugins.nvim-lspconfig</li>
<li>vimPlugins.nvim-scrollview</li>
<li>vimPlugins.nvim-treesitter</li>
<li>vimPlugins.syntastic</li>
<li>vimPlugins.telescope-nvim</li>
<li>vimPlugins.vim-airline</li>
<li>vimPlugins.vim-floaterm</li>
<li>vimPlugins.vim-gnupg</li>
<li>vimPlugins.vim-go</li>
<li>vimPlugins.vim-illuminate</li>
<li>vimPlugins.vim-javacomplete2</li>
<li>vimPlugins.vim-lsc</li>
<li>vimPlugins.vim-lsp</li>
<li>vimPlugins.vim-oscyank</li>
<li>vimPlugins.vim-pathogen</li>
<li>vimPlugins.vim-polyglot</li>
<li>vimPlugins.vim-snippets</li>
<li>vimPlugins.vim-test</li>
<li>vimPlugins.vim-vsnip</li>
<li>vimPlugins.vimspector</li>
<li>vimPlugins.vimtex</li>
<li>vimPlugins.yats-vim</li>
</ul>
</details>

**Pull request merged**